sites/docs/pages/deployment/rendering-modes.md

```diff
@@ -59,22 +61,11 @@ import adapter from '@sveltejs/adapter-static';
 
 /** @type {import("@sveltejs/kit").Config} */
 export default {
-    preprocess: [],
-    compilerOptions: {
-        dev: true
-    },
     kit: {
         adapter: adapter({
             fallback: 'index.html'
         })
     },
-    vite: {
-        server: {
-            watch: {
-            usePolling: true
-            }
-        }
-    }
```

Comment:
@archiewood When Brian was showing me how to enable SPA mode, these options weren't mentioned. Is there any reason why you included them? I don't know if they are necessary or not for SPA mode, but they don't seem to be.